### PR TITLE
BHBC-1867: Bug fix +  adjust sql that pulls the user list

### PR DESCRIPTION
**Pull request by anissa-agahchen**
# Overview

## Links to Jira tickets

- https://quartech.atlassian.net/browse/BHBC-1867

### CODE REVIEW

File: api/src/queries/users/user-queries.ts [hunk 1]
```diff
@@ -99,8 +99,12 @@ export const getUserListSQL = (): SQLStatement | null => {
       system_role sr
     ON
       sur.system_role_id = sr.system_role_id
+    LEFT JOIN
+    	user_identity_source uis
+    ON
+    	su.user_identity_source_id = uis.user_identity_source_id
     WHERE
-      su.record_end_date IS NULL
+      su.record_end_date IS NULL and uis.name not in('DATABASE')
```
Comment (NickPhura) [inline api/src/queries/users/user-queries.ts]: can you swap out the hardcoded 'DATABASE' with the enum SYSTEM_IDENTITY_SOURCE.DATABASE

**Review by curtisupshall — DISMISSED** [review]
🐛

**Review by AlfredRosenthal — DISMISSED** [review]
👍

**Review by AlfredRosenthal — APPROVED** [review]
looks good

**Pull request merged**